app/controllers/items_controller.rb

```diff
@@ -600,7 +595,8 @@ def create_obj
 
   def save_and_reindex
     @object.save
```

Comment:
save is going to call to_solr underneath anyway, right?  This is doubling it up?